frontend/src/container/FeedItemWithPreview.jsx

```diff
@@ -81,6 +81,25 @@ export class FeedItemWithPreview extends React.Component {
     this.props.appContentRemoveCommentAsFile(fileToRemove, this.setState.bind(this))
   }
 
+  handleClickEditComment = (comment) => {
+    const { props } = this
+    props.appContentEditComment(
```

Comment:
Will this work in the recent activities? As we are not in an app there?

Reply:
We don't have comments at recent activities